### PR TITLE
Adding README for openshift_manageiq

### PR DESCRIPTION
**Pull request by enoodle**
I hope this will make this role more clear.

### CODE REVIEW

**Comment by enoodle** [comment]
@tbielawa please review

**Comment by sdodson** [comment]
LGTM, @tbielawa ping me to bypass ci tests if you approve.

File: roles/openshift_manageiq/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ManageIQ
+
+Allows ManageIQ to manage the Openshift cluster.
+This role sets up the `"management-infra"` namespace with the management-admin and inspector-admin service accounts.
+
+###Required vars:
```
Comment (tbielawa) [inline roles/openshift_manageiq/README.md]: These headers do not format correctly without content

Comment (tbielawa) [inline roles/openshift_manageiq/README.md]: screenshot from 2017-08-07 09-12-44

Comment (enoodle) [inline roles/openshift_manageiq/README.md]: @tbielawa :+1: I removed them

**Comment by tbielawa** [comment]
@sdodson let's do this thing

**Pull request merged**